students/noahjones

```diff
@@ -0,0 +1,351 @@
+            background: ;
+        }
+        
+        .face {
+            position: relative;
+            width: 300px;
+            height: 300px;
+            border-radius: 50%;
+            background: #ffcc09;
+            justify-content: center;
+            align-items: center;
+        }
+        
+        .face::before {
+            content: '';
+            position: absolute;
+            top: 180px;
+            width: 150px;
+            height: 70px;
+            background: #b57700;
+            left: 74.3px;
+            border-bottom-left-radius: 70px;
+            border-bottom-right-radius: 70px;
+            transition: 0.5s;
+        }
+        
+        .face:hover::before {
+            top: 210px;
+            width: 150px;
+            height: 20px;
+            background: #b57700;
+            left: 74.3px;
+            border-bottom-left-radius: 0px;
+            border-bottom-right-radius: 0px;
+        }
+        
+        .eyes {
+            position: relative;
+            top: 58px;
+            left: 40px;
+            display: flex;
+        }
+        
+        .eyes .eye {
+            position: relative;
+            width: 80px;
+            height: 80px;
+            display: block;
+            background: #fff;
+            margin: 0 15px;
+            border-radius: 50%;
+        }
+        
+        .eyes .eye::before {
+            content: '';
+            position: absolute;
+            top: 50%;
+            left: 25px;
+            transform: translate(-50%, -50%);
+            width: 40px;
+            height: 40px;
+            background: #333;
+            border-radius: 50%;
+        }
+    </style>
+
+    <title>NJ Jones Tech Profile Page</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="m">
+            J
+        </div>
+        <div class="e">
+            O
+        </div>
+        <div class="g">
+            N
+        </div>
+        <div class="a">
+            E
+        </div>
+        <div class="n">
+            S
+        </div>
+        <div class="ex">
+            !
+        </div>
+
+    </div>
+
+    <body>
+        <div class="face">
+            <div class="eyes">
+                <div class="eye"></div>
+                <div class="eye"></div>
+            </div>
+        </div>
+    </body>
+    <script>
+        document.querySelector('body').addEventListener('mousemove', eyeball);
+
+        function eyeball() {
+            var eye = document.querySelectorAll('.eye');
+            eye.forEach(function(eye) {
+                let x = (eye.getBoundingClientRect().left) + (eye.clientWidth / 2);
+                let y = (eye.getBoundingClientRect().top) + (eye.clientHeight / 2);
+                let radian = Math.atan2(event.pageX - x, event.pageY - y);
+                let rot = (radian * (180 / Math.PI) * -1) + 270;
+                eye.style.transform = "rotate(" + rot + "deg)"
+            })
+        }
+    </script>
+    <center>
```

Comment:
🔧 The [`<center>` tag is obsolete](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center). 
We probably want to change this tag for a `<section>` with a class so that you can center it.